functions/go/create-setters/README.md

```diff
@@ -0,0 +1,135 @@
+# create-setters
+
+### Overview
+
+<!--mdtogo:Short-->
+
+Add setter comments to matching resource fields. Setters serve as parameters for template-free setting of comments.
+
+<!--mdtogo-->
+
+### Synopsis
+
+<!--mdtogo:Long-->
+
+Setters are a safer alternative to other substitution techniques which do not
+have the context of the structured data. Setter comments can be added to 
+parameterize the field values of resources using create-setters function.
+
+We use ConfigMap to configure the `create-setters` function. The desired setter
+values are provided as key-value pairs using `data` field.
+Here, the key is the name of the setter which is used to set the comment and value is the field value to parameterize.
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: create-setters-fn-config
+data:
+  setter_name1: setter_value1
+  setter_name2: setter_value2
+```
+
+<!--mdtogo-->
+
+### Examples
+
+<!--mdtogo:Examples-->
+
+#### Setting comment for scalar values
+
+Let's start with the input resource in a package
+
+```yaml
+apiVersion: v1
+kind: Deployment
+metadata:
+  name: ubuntu-deployment 
+spec:
+  replicas: 3 
+```
+
+Declare the name of the setter with the value for which comments should be added.
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: create-setters-fn-config
+data:
+  image: ubuntu
+  replicas: "3"
+```
+
+Render the declared values by invoking:
+
+```sh
+$ kpt fn eval --image gcr.io/kpt-fn/create-setters:unstable --fn-config ./create-setters-fn-config
+```
+
+Alternatively, setter values can be passed as key-value pairs in the CLI
+
+```sh
+$ kpt fn eval --image gcr.io/kpt-fn/create-setters:unstable -- 'image=ubuntu' 'replicas=3'
```

Comment:
you don't need the quotes here. You would only need the quotes to prevent the processing a value